### PR TITLE
Refs 2048: Update popular repositories ouiaID's for QE tests

### PR DESCRIPTION
**Pull request by Andrewgdewar**

### CODE REVIEW

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx [hunk 1]
```diff
@@ -390,15 +390,15 @@ const PopularRepositoriesTable = () => {
                         <Dropdown
                           onSelect={onDropdownSelect}
                           className={classes.addRepositoriesButton}
-                          ouiaId='add-selected-toggle-dropdown'
+                          ouiaId='add-selected-dropdown'
```
Comment (swadeley) [inline src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx:393]: lets revert this change to avoid partial match. The original will work fine with the other changes.

**Review by swadeley — APPROVED** [review]
OK

File: src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx (hunk 1 above)
Comment (swadeley) [inline src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx:393]:
```suggestion
                          ouiaId='add-selected-toggle-dropdown'
```

**Comment by swadeley** [comment]
Thank you
